<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntersectionHandler</class>
 <widget class="QWidget" name="IntersectionHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="feedbackBox">
     <property name="text">
      <string>Обратная связь</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Автовозврат</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Длительность</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="durationSpin">
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="modelEdit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="nameEdit">
     <property name="placeholderText">
      <string>Введите имя анимации</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addKeyButt">
     <property name="text">
      <string>Добавить точку</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="upButt">
     <property name="text">
      <string>Выше по дереву</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="downButt">
     <property name="text">
      <string>Ниже по дереву</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetButt">
     <property name="text">
      <string>Сбросить выбор</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="baseButt">
     <property name="text">
      <string>Базовая модель</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="x2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="illumEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="zr1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="z1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="zr2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QRadioButton" name="alphaButt">
       <property name="text">
        <string>Альфа</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="y2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="moveButt">
       <property name="text">
        <string>Перемещение</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QDoubleSpinBox" name="illum1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="x1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="yr2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="y1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="z2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="illumButt">
       <property name="text">
        <string>Освещение</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="rotButt">
       <property name="text">
        <string>Вращение</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QDoubleSpinBox" name="xr2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QDoubleSpinBox" name="alpha1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="yr1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QDoubleSpinBox" name="xr1">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QDoubleSpinBox" name="alpha2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QDoubleSpinBox" name="illum2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addButt">
     <property name="text">
      <string>Создать анимацию для выбранного узла</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="startButt">
     <property name="text">
      <string>Запустить анимацию</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>moveButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>x1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>y1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>z1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>xr1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>yr1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>zr1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>illumButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>illum1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>illumButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>illumEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alphaButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>alpha1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>x2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alphaButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>alpha2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>illumButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>illum2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>zr2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>yr2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>xr2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>z2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveButt</sender>
   <signal>toggled(bool)</signal>
   <receiver>y2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>feedbackBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
